src/kARAN1595/BankAccount.java

```diff
@@ -0,0 +1,264 @@
+/*Assignment 11 :  19th Jun 2020
+
+	A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+	          a) Person can debit or credit amount from respective bank.
+	              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+	          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+	          c) A Method to show total credit, debit operations 
+	                 output : Total credit operation - 7 times
+	                              Total debit operation - 5 times
+
+	         d) A Method to show individual bank credit operations.
+	                output : ICICI bank Credit Operation - 2
+	                              HDFC bank Credit Operation - 2
+			              SBI bank Credit Operation - 1
+	                              BOB bank Credit Operation - 4
+
+	         e) A Method to show individual bank debit operations.
+	                output : ICICI bank Debit Operation - 2
+	                             HDFC bank Debit Operation - 2
+			             SBI bank Debit Operation - 1
+	                             BOB bank Debit Operation - 4
+
+	         f) A Method which should return "*I appreciate your money management skill*" message if credit operation count is more than debit operation count else return "*Please spend money wisely*" and print this message in main method.*/
+
+
+
+package kARAN1595;
+
+public class BankAccount {
+
+	
+	
+	int iciciCreditCount,iciciDebitCount;
+	int hdfcCreditCount,hdfcDebitCount;
+	int sbiCreditCount,sbiDebitCount;
+	int bobCreditCount,bobDebitCount;
+	int totalCreditCount,totalDebitCount;
```

Comment:
For this two-variable you should create static variable. Because right now you are performing an action using one reference variable (means for one user) what about if you want to perform for 10 users.